tests/codeception/acceptance/StanfordImageCTACest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/**
+ * Codeception tests on Image CTA paragraph type.
+ */
+
+use Drupal\media\Entity\Media;
+
+/**
+ *
+ */
+class StanfordImageCTACest {
+
+  /**
+   *
+   */
+  protected function setupContentType(\AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/structure/types/manage/page/fields/add-field');
+    $I->selectOption('#edit-new-storage-type', 'Paragraph');
+    $I->selectOption('#edit-new-storage-type', 'Paragraph');
+    $I->seeElement('#edit-label');
+    $I->fillField('#edit-label', 'Paragraphs');
+    $I->click('//input[@class="button button--primary js-form-submit form-submit"]');
+    $I->seeElement('#edit-field-name');
+    $I->fillField('#edit-field-name', 'paragraphs');
+    $I->click('//input[@class="button button--primary js-form-submit form-submit"]');
+    $I->seeInCurrentUrl('node.page.field_paragraphs');
+    $I->click('//input[@id="edit-submit"]');
+    $I->seeElement('#edit-settings-handler-settings-negate-1');
+    $I->click('#edit-settings-handler-settings-negate-1');
+    $I->click('//input[@id="edit-submit"]');
+    drupal_flush_all_caches();
+  }
+
+  /**
+   *
+   */
+  protected function revertContentType(\AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/admin/structure/types/manage/page/fields/node.page.field_paragraphs/delete');
+    $I->click('//input[@id="edit-submit"]');
+    drupal_flush_all_caches();
+  }
+
+  /**
+   *
+   */
+  protected function setupImage() {
+    // Write our base64 image to the data directory.
+    $file = fopen(codecept_data_dir() . '/test_image1.jpg', 'wb');
+    fwrite($file, base64_decode($this->testImage()));
+    fclose($file);
+    $file_data = file_get_contents(codecept_data_dir() . '/test_image1.jpg');
+    $uploaded = file_save_data($file_data, 'public://test_image1.jpg', FILE_EXISTS_REPLACE);
+    $media = Media::create([
+      'bundle' => 'image',
+      'uid' => \Drupal::currentUser()->id(),
+      'field_media_file' => [
+        'target_id' => $uploaded->id(),
+      ],
+    ]);
+    $media->setName('Test Image')->setPublished(TRUE)->save();
+    return $media;
+  }
+
+  /**
+   *
+   */
+  protected function testImage() {
```

Comment:
put the png or jpg in the test directory and just copy it instead of having this blob here.

Reply:
done.